pwndbg/commands/vmmap.py

```diff
@@ -29,6 +29,13 @@ def pages_filter(gdbval_or_str):
     else:
         raise argparse.ArgumentTypeError("Unknown vmmap argument type.")
 
+def print_vmmap_table_header():
+    """
+    Prints the table header for the vmmap command.
+    """
+    width = 2 + 2 * pwndbg.gdblib.arch.ptrsize
+    fmt_string = "%#{}s %#{}s %#4s %#8s %#6s %s".format(width, width)
+    print(fmt_string % ("Start", "End", "Perm", "Size", "Offset", "File"))
```

Comment:
nitpick: we can probably do all this in the previous format() call, right? If so, we should probably just do that.

Reply:
we can probably use f-string literal for all this

Reply:
This didn't get fixed 😞

Reply:
I know but I don't think it is that big deal; it was also copied from another part of the code ;p

we can easily fix it too :P